src/components/ArticleDetails.jsx

```diff
@@ -34,13 +43,23 @@ const ArticleDetails = () => {
 	return (
 		<section className="article-view-container">
 			<h1 id="article-title">{singleArticle.title}</h1>
-			<h2 id="article-author">Written by: {singleArticle.author}</h2>
+			<h3 id="article-author">By: {singleArticle.author}</h3>
 			<img src={singleArticle.article_img_url}></img>
-			<p>Created: {formattedDate}</p>
-			<p>Votes: {singleArticle.votes}</p>
-			<p>Topic: {singleArticle.topic}</p>
-			<p>Comments: {singleArticle.comment_count}</p>
-			<p>{singleArticle.body}</p>
+			<div className="article-details-box">
+				<p id="article-topic">Topic: {singleArticle.topic}</p>
+				<p id="article-date">Created: {formattedDate}</p>
+				<p id="article-body">{singleArticle.body}</p>
+				<p id="article-votes">Votes: {singleArticle.votes}</p>
+			</div>
+			<div className="comments">
```

Comment:
would be a good separation of concerns to have the comments in its own component - that way you could also separate thee logic of handling comment loading too.